test/fixtures/db-stop.js

```diff
@@ -27,4 +75,581 @@ module.exports = {
 		taxi: false
 	},
 	transitAuthority: 'VBB',
+	ids: {
```

Comment:
Wouldn't these be `station.localId` and `station.dhid` instead of `station.ids.…`? (same below)

Reply:
This should have been added as part of #163, which is already on `master`.